src/redux/session.js

```diff
@@ -142,6 +142,8 @@ module.exports.selectIsSocial = state => get(state, ['session', 'session', 'perm
 module.exports.selectIsEducator = state => get(state, ['session', 'session', 'permissions', 'educator'], false);
 module.exports.selectProjectCommentsGloballyEnabled = state =>
     get(state, ['session', 'session', 'flags', 'project_comments_enabled'], false);
+module.exports.selectStudioCommentsGloballyEnabled = state =>
```

Comment:
@picklesrus I think that this should probably be true by default right until we hear from the session? I've been trying to make sure the path to trying to load stuff is as parallel as possible, and this change would prevent loading the comments list until the user session has been fetched, which isn't necessary.